backend/src/app.py

```diff
@@ -71,16 +70,16 @@ def index(n):
     with indexer:
         indexer.index(papers, request_size=32)
 
+if __name__ == '__main__':
```

Comment:
We only really need this if we want to run this file both as a module and as a script. As of now, this would only be a script. I generally lean towards not using this construct, but I don't have a strong opinion on this.